packages/providers/src/operations.graphql

```diff
@@ -66,8 +62,6 @@ fragment receiptFragment on Receipt {
   reason
   ra
   rb
-  rc
-  rd
```

Comment:
I think you should keep `rc` and `rd` because of they're used in `assembleReceiptByType()`:
https://github.com/FuelLabs/fuels-ts/blob/cfb06e88dba97792b7bc5f4e048c9b9373fb3c03/packages/providers/src/utils/receipts.ts#L146-L147

Reply:
@arboleya Yeah, but this is expected because `assembleRceiptByType` was made to take into account all properties existent on receipts. 

But since these 2 values were not being used anywhere else I presumed that they could be removed.

But I reckon that me most critical ones were the contract bytecodes.

Reply:
Removing these props from the GQL may be ok, but we must also purge all remaining usages.